packages/ai-chat/src/chat/components-legacy/input/Input.tsx

```diff
@@ -504,7 +504,7 @@ function Input(props: InputProps, ref: Ref<InputFunctions>) {
           </div>
 
           <div className="cds-aichat--input-container__send-button-container">
-            {isStopStreamingButtonVisible && (
+            {(true || isStopStreamingButtonVisible) && (
```

Comment:
```suggestion
            {isStopStreamingButtonVisible && (
```

we might want to revert this?
the `stop response` button always renders in the deploy preview.